libtiledbsoma/src/utils/parallel_functions.h

```diff
@@ -0,0 +1,406 @@
+/**
```

Comment:
I believe this is simply a copy from core of known-good code -- if so I won't even peek at it